code/__HELPERS/sanitize_values.dm

```diff
@@ -41,7 +41,7 @@
 
 //more specialised stuff
 /// Return `gender` if it is a valid gender, otherwise `default`. No I did not mean to offend you. -qwerty
-/proc/sanitize_gender(gender,neuter=0,plural=0, default="male")
+/proc/sanitize_gender(gender,neuter=0,plural=1, default="male")
```

Comment:
Because for some reason plural needs to be enabled or it will return a value of default, which in this case is `"male"`

I could alter it so that it checks for `if(MALE, FEMALE, PLURAL)` which then returns the gender?

Reply:
Reverted to the previous version and then moved the plural thing into the if(MALE, FEMALE)